create_builder.go

```diff
@@ -303,6 +303,10 @@ func validateBuildpack(bp dist.Buildpack, source, expectedID, expectedBPVersion
 	return nil
 }
 
-func uriFromLifecycleVersion(version semver.Version) string {
+func uriFromLifecycleVersion(version semver.Version, os string) string {
+	if os == "windows" {
```

Comment:
Given that the only place the two urls differ is in the name of the OS, can we assign the download OS to either 'windows' or 'linux', and then `sprintf` that into the download url?

Reply:
The string being passed in there is being parsed from another object, not selected from a set of constants. I'm fine with whatever; the only benefit of the code above that I'm thinking is protection from an unsupported return value (ex: `centos`, `mac`, `Windows`, `LINUX`, etc).